nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -257,6 +257,23 @@ in {
         '';
       };
     };
+    autoUpdateApps = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Run a auto update of all installed apps from the nextcloud repository.
+        '';
+      };
+      startAt = mkOption {
+        type = with types; either str (listOf str);
+        default = "05:00:00";
+        example = "Sun 14:00:00";
+        description = ''
+          When to run the update. See `systemd.services.<name>.startAt`.
```

Comment:
Seems the `<name>` doesn't get escaped when generating manpages:

```
⇒  nix-build nixos/release.nix -A manpages.x86_64-linux
these derivations will be built:
  /nix/store/mdc1snp1km5y963xc8xw8s5x6w39c3iq-options-db.xml.drv
  /nix/store/6p294xw0frkajlj27dk20wa732pyx4ps-generated-docbook.drv
  /nix/store/729zam31kvlb7b9267mfbrqm99df0vr9-nixos-manual-combined.drv
  /nix/store/xl8vg8w0ykf4g7pnkpsk6wpy4iqksfmg-manual-olinkdb.drv
  /nix/store/w2y87icgfnykcxly87kyxfxcpkp0izj9-nixos-manpages.drv
building '/nix/store/mdc1snp1km5y963xc8xw8s5x6w39c3iq-options-db.xml.drv'...
intermediate.xml:30885: parser error : Opening and ending tag mismatch: name line 30884 and para
</para></nixos:option-description><para><emphasis>Type:</emphasis> string or lis
       ^
intermediate.xml:30885: parser error : Opening and ending tag mismatch: para line 30884 and nixos:option-description
</para></nixos:option-description><para><emphasis>Type:</emphasis> string or lis
                                  ^
intermediate.xml:30891: parser error : Opening and ending tag mismatch: option-description line 30884 and listitem
            </filename></member></simplelist></listitem></varlistentry><varliste
                                                        ^
intermediate.xml:30891: parser error : Opening and ending tag mismatch: listitem line 30884 and varlistentry
            </filename></member></simplelist></listitem></varlistentry><varliste
                                                                       ^
intermediate.xml:59111: parser error : Opening and ending tag mismatch: varlistentry line 30884 and variablelist
      </filename></member></simplelist></listitem></varlistentry></variablelist>
                                                                               ^
intermediate.xml:59111: parser error : Opening and ending tag mismatch: variablelist line 3 and appendix
lename></member></simplelist></listitem></varlistentry></variablelist></appendix
                                                                               ^
intermediate.xml:59113: parser error : Premature end of data in tag appendix line 3

^
unable to parse intermediate.xml
note: keeping build directory '/tmp/nix-build-options-db.xml.drv-1'
builder for '/nix/store/mdc1snp1km5y963xc8xw8s5x6w39c3iq-options-db.xml.drv' failed with exit code 6
cannot build derivation '/nix/store/6p294xw0frkajlj27dk20wa732pyx4ps-generated-docbook.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/729zam31kvlb7b9267mfbrqm99df0vr9-nixos-manual-combined.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/w2y87icgfnykcxly87kyxfxcpkp0izj9-nixos-manpages.drv': 1 dependencies couldn't be built
error: build of '/nix/store/w2y87icgfnykcxly87kyxfxcpkp0izj9-nixos-manpages.drv' failed
```

Looking at intermediate.xml:

```
            </filename></member></simplelist></listitem></varlistentry><varlistentry><term xlink:href="#opt-services.nextcloud.autoUpdateApps.startAt" xml:id="opt-services.nextcloud.autoUpdateApps.startAt"><option>services.nextcloud.autoUpdateApps.startAt</option></term><listitem><nixos:option-description><para>When to run the update. See `systemd.services.<name>.startAt`.
```

Docbook inserts an unescaped `<name>` tag

Can you change this to ` When to run the update. See `systemd.services.&lt;name&gt;.startAt`.` to workaround that for now?

Reply:
For completeness, I opened https://github.com/NixOS/nixpkgs/issues/61800, but I guess for now, escaping on your side for now should be fine :-)